variables.tf

```diff
@@ -395,6 +402,32 @@ variable "transit_encryption_enabled" {
   EOT
 }
 
+# Serverless-specific variables
+
+
+variable "name" {
+  description = "Name for serverless cache."
+  type        = string
+  default     = null
+}
+
+variable "data_storage_maximum" {
+  description = "Maximum data storage for serverless cache."
+  type        = number
+  default     = 50
+}
+
+variable "ecpu_per_second_maximum" {
+  description = "Maximum ECPU units per second for serverless cache."
+  type        = number
+  default     = 10000
+}
+
+variable "serverless_tags" {
+  description = "Tags for serverless cache."
+  type        = map(any)
+}
+
```

Comment:
* It's easier and better to take variables description from **[Documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elasticache_serverless_cache).**
 
 * Format should be : 
```
 type
 default
 description
```